packages/runtime/src/next/components/head-tags.tsx

```diff
@@ -0,0 +1,66 @@
+import { ComponentPropsWithoutRef } from 'react'
+import { useIsPagesRouter } from '../../components/hooks/useIsPagesRouter'
```

Comment:
We probably want this hook to be under `src/next` as well. Also, as far as naming conventions, we should use `kebab-case.ts`